pyomo/contrib/pynumero/sparse/warnings.py

```diff
@@ -0,0 +1,4 @@
+
+
+class MPISpaceWarning(Warning):
```

Comment:
Was this used anywhere?

Reply:
Good question. I don't think so. I'll remove it.